src/scss/components/topnav/index.scss

```diff
@@ -161,13 +178,12 @@
     }
 
     hx-menuitem:not(.hxMenuKey):hover {
-      background-color: $gray-900;
-      color: $gray-0 !important;
+      color: #970c8d !important;
```

Comment:
Please change color from `#970c8d` to `$purple-500`.

Reply:
Changes have been done, please review.